docs/src/joint/api/dia/Paper/prototype/hasScheduledUpdates.html

```diff
@@ -0,0 +1,2 @@
+<pre class="docs-method-signature"><code>paper.hasScheduledUpdates()</code></pre>
+<p>Return <code>true</code> if there were any changes made to the graph, that still need to be updated in the paper (update views). Return <code>false</code> otherwise.</p>
```

Comment:
```suggestion
<p>Return <code>true</code> if any changes were made to the graph which are not yet reflected in the paper. (Call the <code>updateViews</code> <a href="#dia.Paper.prototype.updateViews">function</a> to update these views.) Return <code>false</code> otherwise.</p>
```